### ISSUE TITLE
10 minute seek with SHIFT+PgUp does not work with audio files

### ISSUE THREAD

**Issue by bgx90** [mpv-player/mpv#7310]
mpv does not seek 10 minutes forward or backward in audio-only files (mp3, m4a, etc.) when SHIFT+PgUp is pressed. I have confirmed that 10 minute seek works in mplayer for all audio files I tested with mpv.

### mpv version and platform

OS: Debian Testing
mpv version:
mpv 0.30.0 Copyright © 2000-2019 mpv/MPlayer/mplayer2 projects
 built on UNKNOWN
ffmpeg library versions:
   libavutil       56.31.100
   libavcodec      58.54.100
   libavformat     58.29.100
   libswscale      5.5.100
   libavfilter     7.57.100
   libswresample   3.5.100
ffmpeg version: 4.2.1-2+b1

### Reproduction steps

1) Open an audio-only file (mp3, m4a, etc.)  that is more than 10 minutes long with mpv
2) Press SHIFT+PgUp

### Expected behavior

mpv seeks 10 minutes forward

### Actual behavior

nothing happens

### Log file

I opened an mp3 that is 16 minutes and 53 seconds long, pressed SHIFT+PgUp, and exited mpv after mpv failed to seek forward 10 minutes.

Here is the log file:
[mpv_log.txt](https://github.com/mpv-player/mpv/files/4010379/mpv_log.txt)

**Comment by wm4** [comment]
Works for me. Your log doesn't show a seek command.

**Comment by wm4** [comment]
PS: and I suspect you're trying this on the console.

**Comment by bgx90** [comment]
Yes, I am using mpv from the terminal. 10 minute seek works for video files, but not for audio-only files.

**Comment by bgx90** [comment]
What version of mpv are you using?  Debian Testing is not using the latest version.

**Comment by wm4** [comment]
Shift+pageup will probably not correctly work on the terminal (because the terminal is a mess from the 80ies), only on the mpv x11/whatever window.

**Comment by bgx90** [comment]
I see. Thank you for explaining.

**Comment by wm4** [comment]
If that's the problem, it could probably be fixed; most terminals map these keys to scrolling the backlog though.

**Comment by bgx90** [comment]
The terminal is indeed the problem. I am using the suckless.org st terminal; it does not have scrollback, but apparently catches the SHIFT+PgUp key press anyway. Since st does not have scrollback, nothing happens on the screen when I press SHIFT+PgUp. I tried running mpv with the xfce terminal which does have scrollback enabled and saw the terminal scroll back when I pressed SHIFT+PgUp.

It would be nice if mpv could be made to use its default key mappings in the terminal to avoid confusion. Of course,  I can always change mpv's mappings in my input.conf file.
